site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -26,12 +26,26 @@ much of the `Fleet` is  incremented and decremented at one time:
       maxUnavailable: 25%
 ```
 
-So when a Fleet is edited, either through `kubectl` `edit`/`apply` or via the Kubernetes API, this performs the following operations:
+So when a Fleet is edited (any field other than `replicas`, see note below), either through `kubectl` `edit`/`apply` or via the Kubernetes API, this performs the following operations:
 
 1. Adds the `maxSurge` number of `GameServers` to the Fleet.
 1. Shutdown the `maxUnavailable` number of `GameServers` in the Fleet, skipping `Allocated` `GameServers`.
 1. Repeat above steps until all the previous `GameServer` configurations have been `Shutdown` and deleted.
 
+{{< alert title="Note" color="info">}}
+When `Fleet` update contains only changes to the `replicas` parameter, then new GameServers will be created/deleted straight away,
+which means in that case `maxSurge` and `maxUnavailable` parameters for a RollingUpdate will not be used.
+The RollingUpdate strategy takes place when you update `spec` parameters other than `replicas`.
+
+For the case when you want to be flexible on Fleet size, Fleetautoscaler and your prefer `kubectl apply` Fleet config with omitted `replicas` parameter should be used.
```

Comment:
Not sure I understand this sentence (and maybe the next one)

Could this be something more like:

```markdown
If you are using a Fleet which is scaled by a FleetAutoscaler, [read the Fleetautoscaler guide]({{< relref "../Getting Started/create-fleetautoscaler.md#7-change-autoscaling-parameters" >}}) for more details on how RollingUpdates with FleetAutoscalers need to implemented.
```
Rather than writing the same documentation twice?
(If I'm hitting the idea of what you were trying to say)